test/e2e/specs/editor/blocks/list.spec.js

```diff
@@ -0,0 +1,872 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'List', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'can be created by using an asterisk at the start of a paragraph block', async ( {
+		editor,
+		page,
+	} ) => {
+		// Create a block with some text that will trigger a list creation.
+		await page.locator( 'role=button[name="Add default block"i]' ).click();
```

Comment:
Nit:

```suggestion
		await page.click( 'role=button[name="Add default block"i]' );
```

Same for the other places below.